### PR TITLE
add dialog function

### PR DESCRIPTION
**Pull request by harshmanek**
added dialog tab in modular_ui

### CODE REVIEW

**Comment by harshmanek** [comment]
hope my first try went well

File: .vscode/settings.json
Comment (omjogani) [inline .vscode/settings.json:1]: Remove this file by adding it to `.gitignore`

**Review by omjogani — CHANGES_REQUESTED** [review]
Please remove` .vscode` folder from the PR. Other things seems good to me!

**Review by omjogani — APPROVED** [review]
LGTM!

**Comment by omjogani** [comment]
Thanks for your contribution @hrxmanek 🚀

**Comment by harshmanek** [comment]
> Thanks for your contribution @hrxmanek 🚀

thankful for the opportunity 😊👍

**Pull request merged**